src/renderer/pages/Staking/Overview/components/NominatorsList/NominatorItem/NominatorItem.tsx

```diff
@@ -16,47 +15,38 @@ type Props = {
   isStakingLoading: boolean;
   stake: NominatorInfo;
   content: ReactNode;
+  addressPrefix?: number;
   onToggleNominator: (nominator: Address, boolean: boolean) => void;
   onCheckValidators: (stash?: Address) => void;
 };
 
 export const NominatorsItem = ({
   nominatorsLength,
   asset,
-  explorers,
+  explorers = [],
   stake,
   content,
   isStakingLoading,
+  addressPrefix,
   onToggleNominator,
   onCheckValidators,
 }: Props) => {
   const { t } = useI18n();
 
   const activeWallet = useUnit(walletModel.$activeWallet);
 
-  const getExplorers = (address: Address, stash?: Address, explorers: Explorer[] = []) => {
-    const explorersContent = explorers.map((explorer) => ({
-      id: explorer.name,
-      value: <ExplorerLink name={explorer.name} href={getAccountExplorer(explorer, { address })} />,
-    }));
-
-    if (!stash) return [{ items: explorersContent }];
-
-    const validatorsButton = (
-      <button
-        type="button"
-        className="flex items-center gap-x-2 px-2 w-full h-full"
-        onClick={() => onCheckValidators(stash)}
-      >
-        <Icon name="viewValidators" size={16} />
-        <FootnoteText as="span" className="text-text-primary">
-          {t('staking.overview.viewValidatorsOption')}
-        </FootnoteText>
-      </button>
-    );
-
-    return [{ items: [{ id: '0', value: validatorsButton }] }, { items: explorersContent }];
-  };
+  const validatorsButton = (
```

Comment:
I would suggest to move button html into jsx, because it's not used anywhere else not sure this variable is worth it